torchtext/data/__init__.py

```diff
@@ -4,8 +4,9 @@
 from .field import RawField, Field, ReversibleField, SubwordField, NestedField, LabelField
 from .iterator import (batch, BucketIterator, Iterator, BPTTIterator,
                        pool)
+from .metrics import bleu_score
 from .pipeline import Pipeline
-from .utils import get_tokenizer, interleave_keys
+from .utils import get_tokenizer, interleave_keys, ngrams_iterator
```

Comment:
@zhangguanheng66 do I actually need to import `ngrams_iterator` here ?

Reply:
Yes, you could.